pkg/webhooks/subscription/subscription.go

```diff
@@ -0,0 +1,183 @@
+package subscription
+
+import (
+	"encoding/json"
+	"net/http"
+	"sync"
+
+	responsehelper "github.com/openshift/managed-cluster-validating-webhooks/pkg/helpers"
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+	"k8s.io/api/admission/v1beta1"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	admissionctl "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	WebhookName    string = "subscription-validation"
+	LoggingSubName string = "cluster-logging"
+	ESSubName      string = "elasticsearch-operator"
+	BlockedChannel string = "4.5"
+)
+
+var (
+	privilegedUsers = []string{"kube:admin", "system:admin"}
+	adminGroups     = []string{"osd-sre-admins", "osd-sre-cluster-admins"}
```

Comment:
What if we took a different approach with this hook? We know that  customers will have specific group membership and so instead of saying that members of these lists are _allowed_, we could say that members of those customer groups are _disallowed_ and everyone else is allowed.

The default case will be (RBAC-based) allow unless the requestor is a member of a customer group trying to make changes.

Reply:
IMO it's simpler to manage access the way it's coded now, allowing only privileged users to do it. 

Doing the opposite and Identifying customer's groups seems harder to implement.

Reply:
:+1: